extensions/dicom-segmentation/src/utils/drawCanvasCrosshairs.js

```diff
@@ -0,0 +1,56 @@
+import cornerstone from 'cornerstone-core';
+import cornerstoneTools from 'cornerstone-tools';
+
+const { importInternal } = cornerstoneTools;
+const draw = importInternal('drawing/draw');
+const drawLine = importInternal('drawing/drawLine');
+const getNewContext = importInternal('drawing/getNewContext');
+
+export default function _drawCanvasCrosshairs(eventData, center, options) {
```

Comment:
I'm having deja vu here. I guess I see why this function is in both extensions though.

Reply:
Yeah it felt bad, but the extensions should be independent. We should keep this in mind when thinking about v3 and sharing generic code snippets.